site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: service, info, policy, credential, config, heal, docker stable, docker edge
 
   - Minio SDKs:
       - JavaScript Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-js/master/README.md
         Slug: javascript-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | JavaScript Client Quickstart Guide
+        SEO Description: This quickstart guide will show you how to install the client SDK and execute an example JavaScript program.
+        SEO Keywords: JavaScript, JavaScript API, npm install minio
 
       - JavaScript Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-js/master/docs/API.md
         Slug: javascript-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | JavaScript Client API Reference
+        SEO Description: This document contains the detailed example code for the JavaScript Client API
+        SEO Keywords: JavaScript Client, JavaScript API
 
       - Java Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-java/master/README.md
         Slug: java-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Java Client Quickstart Guide
+        SEO Description: The MinIO Java Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Java Client SDK
 
       - Java Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-java/master/docs/API.md
         Slug: java-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Java Client API Reference
+        SEO Description: The MinIO Java Client SDK provides detailed code examples for the Java API
+        SEO Keywords: Java Client SDK, Java Client API
 
       - Python Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/README.md
         Slug: python-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client Quickstart Guide
+        SEO Description: The MinIO Python SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Python SDK, Python Client API
 
       - Python Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/docs/API.md
         Slug: python-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client API Reference
+        SEO Description: The MinIO Python SDK provides detailed code examples for the Java API
```

Comment:
There seems to be a mistake here, we don't provide code examples for `Java` in the `Python Client API reference section`.

Also similar to my other comment, the Python Client API reference provides detailed code examples for the Python SDK (Instead of the Python SDK providing detailed code examples for the Java API)

Reply:
Updated but if there are technical subtleties then I am open to discussion.